static/js/buddy_list.js

```diff
@@ -269,7 +269,13 @@ export class BuddyList extends BuddyListConf {
 
         // Add a fudge factor.
         height += 10;
-
+        
+        if (this.keys.length === 0) {
+            this.container = $(this.container_sel);
+            this.container.append(`<div > ${_('User not found')} </div>`);
```

Comment:
I think you want a string more like "No matching users".  I don't think we want a div without a class here.

Please look at the styling we use for similar widgets in the `ListWidget` class.

Reply:
@timabbott  is it acceptable  if i remove the div from here ?